src/pe_reports/helpers/bulletin/bulletin_generator.py

```diff
@@ -0,0 +1,277 @@
+"""
```

Comment:
This file looks good except you need to delete unnecessary comments like on line 199 and use the LOGGER instead of print statements.

Reply:
Addressed in the following commit:[6dfad1b](https://github.com/cisagov/pe-reports/pull/282/commits/6dfad1b08195be4a60b71de96b227e0488931edc)